.github/workflows/patch-release-build.yml

```diff
@@ -160,7 +160,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
-    needs: [ test, smoke-test, example-distro ]
+    needs: [ prepare-release-branch, test, smoke-test, example-distro ]
```

Comment:
Oops, I think I removed this when I added `test`, `smoke-test` and `example-distro`. are needs not transitive?

Reply:
The dependency itself is but probably not that interpolation syntax which is reading this yaml block. May be able to replace `needs.` below with `jobs.` but not sure

Reply:
oh! yes, I see now, thx